database/seeds/DatabaseSeeder.php

```diff
@@ -17,28 +17,33 @@ public function run()
     {
         // Create 10 campaigns with signups & posts.
         factory(Campaign::class, 10)->create()->each(function (Campaign $campaign) {
-            // Add a "default" action so this functions as expected in the "dev" environment.
```

Comment:
This doesn't seem applicable anymore.. removing the `default` title and using the factory title seems to work as I'd expect it to.